notes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+    <title></title><link rel="stylesheet" href="notes.css">
```

Comment:
These should be on different lines. Also, can you think of a title for your page?